apps/src/applab/redux/data.js

```diff
@@ -20,14 +23,23 @@ const DataState = Record({
 const initialState = new DataState();
 
 export default function (state = initialState, action) {
+  let map;
```

Comment:
`let map` could be declared in the DELETE_TABLE_NAME block scope.

As an aside, one reason I tend to prefer a set of if statements instead of a switch in my reducers, is it allows each one to have it's own block scope. (So that I can have a `const foo` inside one of my handler for ACTION_1 and then have a different `const foo` inside my handler for ACTION_2).